plugins/uast-kotlin-fir/src/org/jetbrains/uast/kotlin/internal/firKotlinInternalUastUtils.kt

```diff
@@ -15,7 +15,7 @@ val firKotlinUastPlugin: UastLanguagePlugin by lz {
         ?: FirKotlinUastLanguagePlugin()
 }
 
-internal val PsiElement.service: FirKotlinUastResolveProviderService
+internal val PsiElement.resolveProviderService: KotlinUastResolveProviderService
```

Comment:
For my point of view getting specific things as extension property for `PsiElement` could be not really clear for the code readers. So I'd prefer a method with the `Project` or maybe `PsiElement` parameter as it usually done in IDEA sources

Reply:
Glad this change regarding service consolidation caught your attention here. This was added in the beginning, assuming service retrieval would be often. The counterpart in FE1.0 UAST is actually inside the plugin implementation: https://github.com/JetBrains/kotlin/blob/master/plugins/uast-kotlin/src/org/jetbrains/uast/kotlin/KotlinUastLanguagePlugin.kt#L78

Since the usage scope is within the plugin, and indeed the current usage in FIR UAST is same, there wasn't a good reason to utilize this. (I.e., I don't know why I did that.) Migrated into the plugin. Also, now abstract `UElement` has a (non-null) service, any further usage (from `UElement`) can benefit from it (I hope).